finesse/gui/hardware_set/dummy_ftsw500.yaml

```diff
@@ -3,6 +3,8 @@ name: "Dummy devices + FTSW500 spectrometer"
 devices:
   stepper_motor:
     class_name: stepper_motor.dummy.DummyStepperMotor
+    params:
```

Comment:
Do we need steps per rotation, here?

Reply:
If we miss it out, the default will be used instead, which should be fine.